lib/src/ui/scaner_screen.dart

```diff
@@ -57,6 +57,8 @@ class ScanerScreenState extends State<ScanerScreen> {
             builder: (context, snapshot) => Column(
               mainAxisAlignment: MainAxisAlignment.end,
               children: [
+                if (bleScanner.state.discoveredDevices.isNotEmpty)
+                  const Text("Device list"),
```

Comment:
`Device list` -> `Devices:`.  fontSize can be higher, 20 I guess. And add some extra space after text `const SizedBox(height: 8),`

Reply:
I will do this

Reply:
We can close this thread - As we discussed Device list would part of app bar